Source/WebCore/dom/ExtensionStyleSheets.cpp

```diff
@@ -73,15 +73,15 @@ CSSStyleSheet* ExtensionStyleSheets::pageUserSheet()
     if (m_pageUserSheet)
         return m_pageUserSheet.get();
     
-    Page* owningPage = m_document.page();
+    Page* owningPage = m_document->page();
     if (!owningPage)
         return 0;
     
     String userSheetText = owningPage->userStyleSheet();
     if (userSheetText.isEmpty())
         return 0;
     
-    m_pageUserSheet = createExtensionsStyleSheet(m_document, m_document.settings().userStyleSheetLocation(), userSheetText, UserStyleLevel::User);
+    m_pageUserSheet = createExtensionsStyleSheet(m_document.get(), m_document->settings().userStyleSheetLocation(), userSheetText, UserStyleLevel::User);
```

Comment:
Seems we should pass a protected document here.

Reply:
Fixed.